tests/resources/Util.robot

```diff
@@ -423,6 +433,7 @@ Run Regression Tests
     ${rc}  ${output}=  Run And Return Rc and Output  curl -sk ${vic-admin}/container-logs.tar.gz | tar tvzf -
     Should Be Equal As Integers  ${rc}  0
     Log  ${output}
+    Should Contain  ${output}  ${container}/output.log
```

Comment:
yup - we've got a bugzilla for general log sanitization. I've asked @caglar10ur to raise a github issue that deals with the general case of logging application data as it starts in the tether.